playwright-e2e/dss/component/table.ts

```diff
@@ -17,6 +19,7 @@ export default class Table {
   private readonly footerCss: string;
   private readonly headerRowCss: string;
   private readonly nth: number;
+  private readonly _paginator: ParticipantsListPaginator;
```

Comment:
Adding paginator functionality in table. I will need to rename `ParticipantsListPaginator` in future because it should be generic and not specific to Participant List page.